### DISCUSSION TITLE
borrowing code

### DISCUSSION THREAD

**Discussion by rebroad** [bitcoinknots/bitcoin#33] [Ideas]
Hi @luke-jr . Sorry, I'm not sure how best to contact you. I was wanting to see if you've already rebased the mempool graphing code to work with bitcoin's master branch, to save me the trouble of doing it, but I wasn't sure where to find where you'd have stored it on github if you had.! Thanks if you can point me to it!

**Comment by luke-jr** [comment]
The most recent I have rebased to at the moment is in my `old_stats_qt-0.21` branch
